### PR TITLE
docs: Add Contributor Over Time graph

### PR DESCRIPTION
**Pull request by juzhiyuan**
## What does this PR do?

This PR aims to add the Contributor Over Time to our repo.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

**Comment by eshellman** [comment]
1. The preceding text probably should be changed , since we pass a thousand contributors a while ago.
2. The graph is neat, but I prefer a link to it, not an embed. and maybe in the intro on README in stead of HOWTO

File: HOWTO.md [hunk 1]
```diff
@@ -10,9 +10,7 @@ Welcome to Free-Programming-Books! We welcome new contributors; even those makin
 * [Youtube - Markdown Crash Course](https://www.youtube.com/watch?v=HUBNt18RFbo)
 
 
-Don't hesitate to ask questions; every contributor started with a first PR. You could be our thousandth! Just to see the Contributor Over Time!
-
-![Contributor over time](https://contributor-graph-api.apiseven.com/contributors-svg?repo=EbookFoundation/free-programming-books)
+Don't hesitate to ask questions; every contributor started with a first PR. Join a team of [two thousand contributors!](https://contributor-graph-api.apiseven.com/contributors-svg?repo=EbookFoundation/free-programming-books)
```
Comment (davorpa) [inline HOWTO.md:13]: And if we avoid depend about translations when counter increments. A text like:

`So... how about becoming part of this booming community?`

Or/and link as a badge in README #5983 

Other graphs about contributors: https://github.com/EbookFoundation/free-programming-books/graphs/contributors

**Comment by juzhiyuan** [comment]
Just noticed 😄 Will update this week.

**Review by davorpa — APPROVED** [review]
LGTM :+1:

**Pull request merged**